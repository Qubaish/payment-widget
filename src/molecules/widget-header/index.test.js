import React from 'react';
import { render } from '@testing-library/react';
import WidgetHeader from './index';

describe('<AmoutText />', () => {
    it('checking snapshot', () => {
        const { asFragment } = render(<WidgetHeader partnerName="Business"/>);
        expect(asFragment()).toMatchSnapshot();
    });
    it('should render a image', () => {
        const { getByTestId } = render(<WidgetHeader partnerName="Business"/>);
        const elem = getByTestId('img');
        expect(elem.name).toBe('Business Logo');
    });
});