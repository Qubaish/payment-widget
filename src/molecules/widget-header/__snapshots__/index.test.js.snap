// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AmoutText /> checking snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-AxhCb gzDXAC"
    data-testid="widget-header"
  >
    <img
      class="sc-AxiKw jwJzJl"
      data-testid="img"
      name="Business Logo"
      src="https://static-exp1.licdn.com/sc/h/79m1qiu8wnezlizb5zn646ata"
    />
    <div
      class="sc-AxgMl jsVOUk"
    >
      <p
        class="sc-AxjAm kZBtGC"
        data-testid="text"
      >
        PAYMENT FOR
      </p>
      <p
        class="sc-AxirZ fqAQvC"
        data-testid="text"
      >
        Business
      </p>
    </div>
  </div>
  <hr
    class="sc-AxhUy klmKvd"
  />
</DocumentFragment>
`;
